### PR TITLE
Add black

### PR DESCRIPTION
**Pull request by inverse**
Okays - It didn't convert `camelCase` to `snakeCase` but it's a step in the right direction for consistency.

What do you think?

### CODE REVIEW

**Comment by ZENALC** [comment]
Honestly, all it looks like black is doing is restyling the parentheses for arguments and replacing double quotes with single quotes. While consistent, I think having single quotes for dict keys is cleaner than double quotes... 

I did some quick digging around black and it looks like they enforce double quotes, so I don't believe we have a setting for using single quotes instead. Anyhow, I don't think black is helping us much here. What do you think?

File: algobot/algorithms.py [hunk 1]
```diff
@@ -81,11 +102,25 @@ def get_sma(data: List[Dict[str, float]], prices: int, parameter: str) -> float:
     :param parameter: Parameter from data dictionary with which to get the simple moving average.
     :return: Simple moving average.
     """
-    return sum([get_data_from_parameter(data=period, parameter=parameter) for period in data]) / prices
-
-
-def get_ema(data: List[dict], prices: int, parameter: str, sma_prices: int = 5, memo: dict = None,
-            desc: bool = False) -> Tuple[float, dict]:
+    return (
+        sum(
```
Comment (ZENALC) [inline algobot/algorithms.py:106]: This formatting does not really help us here. It's even more unreadable lol

File: algobot/__main__.py [hunk 2]
```diff
@@ -321,24 +435,39 @@ def initiate_optimizer(self):
             return
 
         combos = self.configuration.get_optimizer_settings()
-        if combos['strategies'] == {}:
-            create_popup(self, "No strategies found. Make sure you have some strategies for optimization.")
+        if combos["strategies"] == {}:
+            create_popup(
+                self,
```
Comment (ZENALC) [inline algobot/__main__.py:440]: Yeah, I don't think this is honestly that readable... No need to separate in into 4 separate lines imo

File: algobot/assets.py [hunk 3]
```diff
@@ -884,18 +884,25 @@
 \x00\x00\x01\x75\x1f\xa3\x93\x96\
 "
 
-qt_version = [int(v) for v in QtCore.qVersion().split('.')]
+qt_version = [int(v) for v in QtCore.qVersion().split(".")]
```
Comment (ZENALC) [inline algobot/assets.py:887]: We shouldn't really touch this file as it's generated by QT itself

Comment (inverse) [reply algobot/assets.py:887]: Ahh okays - we can ignore it if needed

**Comment by inverse** [comment]
Okays - Ill close it then ;) 

While it is highly opinionated it does enforce consistency ;)

**Pull request closed without merging**